docker-compose.yaml

```diff
@@ -125,7 +125,7 @@ services:
 
   keycloak:
     container_name: keycloak_container
-    image: quay.io/keycloak/keycloak:25.0
```

Comment:
We're not ready to upgrade Keycloak to v26 yet. We've been waiting for a compatible version of https://github.com/adorsys/keycloak-config-cli, which just came out. We'll be doing a careful upgrade in the coming days / weeks to ensure all the edge cases still work, since it is a major version update.